powerstore/datasource_host.go

```diff
@@ -0,0 +1,367 @@
+							Description:         "Operating system of the host.",
+							MarkdownDescription: "Operating system of the host.",
+						},
+						"host_connectivity": schema.StringAttribute{
+							Computed:            true,
+							Description:         "Connectivity type for hosts.",
+							MarkdownDescription: "Connectivity type for hosts.",
+						},
+						"type": schema.StringAttribute{
+							Computed:            true,
+							Description:         "Type of hosts.",
+							MarkdownDescription: "Type of hosts.",
+						},
+						"type_l10n": schema.StringAttribute{
+							Description:         "Localized message string corresponding to type",
+							MarkdownDescription: "Localized message string corresponding to type",
+							Computed:            true,
+						},
+						"os_type_l10n": schema.StringAttribute{
+							Description:         "Localized message string corresponding to OS type.",
+							MarkdownDescription: "Localized message string corresponding to OS type.",
+							Computed:            true,
+						},
+						"host_connectivity_l10n": schema.StringAttribute{
+							Description:         "Localized message string corresponding to host connectivity.",
+							MarkdownDescription: "Localized message string corresponding to host connectivity.",
+							Computed:            true,
+						},
+						"initiators": schema.ListNestedAttribute{
+							Description:         "Initiator instance.",
+							MarkdownDescription: "Initiator instance.",
+							Computed:            true,
+							NestedObject: schema.NestedAttributeObject{
+								Attributes: map[string]schema.Attribute{
+									"port_type": schema.StringAttribute{
+										Description:         "Protocol type of the host initiator.",
+										MarkdownDescription: "Protocol type of the host initiator.",
+										Computed:            true,
+									},
+									"port_name": schema.StringAttribute{
+										Description:         "The port name, one of: IQN, WWN, or NQN..",
+										MarkdownDescription: "The port name, one of: IQN, WWN, or NQN..",
+										Computed:            true,
+									},
+									"chap_mutual_password": schema.StringAttribute{
+										Description:         "Password for CHAP authentication.",
+										MarkdownDescription: "Password for CHAP authentication.",
+										Computed:            true,
+									},
+									"chap_mutual_username": schema.StringAttribute{
+										Description:         "Username for CHAP authentication.",
+										MarkdownDescription: "Username for CHAP authentication.",
+										Computed:            true,
+									},
+									"chap_single_username": schema.StringAttribute{
+										Description:         "Username for CHAP authentication.",
+										MarkdownDescription: "Username for CHAP authentication.",
+										Computed:            true,
+									},
+									"chap_single_password": schema.StringAttribute{
+										Description:         "Password for CHAP authentication.",
```

Comment:
Are we getting password in the response?

Reply:
Passwords were always blank in the response, removed passwords from the schema